h/config.py

```diff
@@ -24,6 +24,7 @@ def settings_from_environment():
     _setup_client(settings)
     _setup_statsd(settings)
     _setup_websocket(settings)
+    _setup_csp(settings)
```

Comment:
The config module has changed substantially in #3035 so this will need rebasing. Looks like the options you've added will map straightforwardly, though.